packages/nimble-components/src/rich-text-editor/specs/README.md

```diff
@@ -48,7 +53,8 @@ The `nimble-rich-text-viewer` provides support for converting the input markdown
 
 -   Allowing the user to tag or mention by entering `@` in the editor and selecting the user name from the drop-down list.
 -   Support for adding images to the editor either by uploading or by pasting it.
--   Support for adding hyperlinks to the existing words in the editor.
+-   Support for adding hyperlinks to the existing text in the editor. This allows users to add links to existing text in the editor. When the
+    link button in the formatting options is clicked, a dialog opens, providing a space to enter the hyperlink for the selected text.
```

Comment:
We are yet to research and explore on the hyperlink dialog implementation strategy. Currently it's added under out of scope section. This should have been captured earlier by us, under out of scope. 

Since this point is added in out of scope now, the diff shows only this point without showing the actual heading of this section.

Hope it should be fine to resolve this comment. Let us know otherwise.

Reply:
Sorry about that! Github had collapsed the header for that section so I thought it was part of the Features plan.